yb-voyager/src/srcdb/ora2pg_extract_schema.go

```diff
@@ -83,13 +54,24 @@ func ora2pgExtractSchema(source *Source, exportDir string) {
 		}
 
 		err = exportSchemaObjectCommand.Wait()
+		if outbuf.String() != "" {
+			log.Infof(`ora2pg STDOUT: "%s"`, outbuf.String())
+		}
+		if errbuf.String() != "" {
+			log.Errorf(`ora2pg STDERR in export of %s : "%s"`, exportObject, errbuf.String())
+		}
 		if err != nil {
 			utils.PrintAndLog("Error while waiting for export command exit: %v", err)
 			exportSchemaObjectCommand.Process.Kill()
```

Comment:
We don't need to explicitly call this `Process.Kill()`. So remove all such occurrences.